internals/secrethub/inject.go

```diff
@@ -61,8 +63,10 @@ func (cmd *InjectCommand) Register(r Registerer) {
 			units.HumanDuration(cmd.clearClipboardAfter),
 		),
 	).Short('c').BoolVar(&cmd.useClipboard)
-	clause.Flag("file", "Write the injected template to a file instead of stdout.").StringVar(&cmd.file)
-	clause.Flag("file-mode", "Set filemode for the file if it does not yet exist. Defaults to 0600 (read and write for current user) and is ignored without the --file flag.").Default("0600").SetValue(&cmd.fileMode)
+	clause.Flag("in-file", "The filename of a template file to inject.").Short('i').StringVar(&cmd.inFile)
+	clause.Flag("out-file", "Write the injected template to a file instead of stdout.").Short('o').StringVar(&cmd.outFile)
+	clause.Flag("file", "").Hidden().StringVar(&cmd.outFile)
```

Comment:
Could you add a comment to this line indicating it's aliased? If someone comes along in 6 months, it needs to be self-evident that this is an alias. 

```
// Alias for --out-file
```

Reply:
We might want to create a `flag.Alias(name string)` function later to make it self-explanatory.